### PR TITLE
[PREVIEW] RDM-3631: Adding Family Public Law content

### PR DESCRIPTION
**Pull request by MKaramath**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-3631

### Change description ###
Adding Family Public Law content within coookies, privacy policies, terms and conditions and contact us

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/app/footer-nav/cookies.component.html [hunk 1]
```diff
@@ -480,8 +480,8 @@ <h3 class="heading-medium">To make the service more secure</h3>
     </div>
   </ccd-accordion>
 
-  <!-- Financial Remedy Accordion stuff... -->
-  <ccd-accordion [title]="'Financial Remedy'" [lastAccordion]="true">
+  <!-- Financial Remedy Accordion stuff -->
+  <ccd-accordion [title]="'Financial Remedy'" [lastAccordion]="false">
     <div class="accordion-body">
```
Comment (adr1ancho) [inline src/app/footer-nav/cookies.component.html]: Why match to false?

File: src/app/footer-nav/tc.component.html [hunk 2]
```diff
@@ -446,4 +446,172 @@ <h3 class="heading-medium">Contact us</h3>
...
+        <ul style="list-style-type: disc;">
+          <li>viruses</li>
+          <li>malicious computer code</li>
+          <li>other forms of interference which may damage your computer system</li>
+        </ul>
+      </div>
+      <div class="para">
+        <span class="text-16">You must not misuse this service by knowingly introducing viruses,trojans, worms,
+          logic bombs or other material which is malicious or technologically harmful. You must not attempt to
+          gain unauthorised access to this service, the system on which it is stored, or any server,computer or
```
Comment (noronhaa) [inline src/app/footer-nav/tc.component.html]: Lets add a whitespace after the comma in  'server,computer'

File: src/app/footer-nav/privacy.component.html [hunk 3]
```diff
@@ -617,4 +617,124 @@ <h3 class="heading-medium">How to complain</h3>
...
+      <!-- Family Public Law Start -->
+      <div class="para">
+        <span class="text-16">This privacy policy explains why we collect your personal data, what we do with it, and
+          your rights. More information about using this service is in the
+          <a [routerLink]="'/terms-and-conditions'" target="_blank">terms and conditions</a></span>
+      </div>
+
+      <h3 class="heading-medium">Who manages this service</h3>
+      <div class="para">
+        <span class="text-16">This service is managed by Her Majesty’sCourts and Tribunals Service (HMCTS) and we’re
```
Comment (noronhaa) [inline src/app/footer-nav/privacy.component.html]: Whitespace needed - 'Majesty’sCourts'

File: src/app/footer-nav/privacy.component.html [hunk 4]
```diff
@@ -617,4 +617,124 @@ <h3 class="heading-medium">How to complain</h3>
...
+      </div>
+
+      <h3 class="heading-medium">What is the personal data that we collect</h3>
+      <div class="para">
+        <span class="text-16">When you use the Family Public Law service, we collect the following personal data:</span>
+        <ul style="list-style-type: disc;">
+          <li>he applicant's name, address, job role and contact details</li>
+          <li>the Local Authority solicitor's name, address and contact details</li>
+          <li>the social worker's name, address and contact details</li>
+          <li>name, date of birth, gender, and address of children in the case</li>
```
Comment (noronhaa) [inline src/app/footer-nav/privacy.component.html]: remove comma after 'gender'

File: src/app/footer-nav/privacy.component.html [hunk 5]
```diff
@@ -617,4 +617,124 @@ <h3 class="heading-medium">How to complain</h3>
...
+      <div class="para">
+        <span class="text-16">We collect personal data to:</span>
+        <ul style="list-style-type: disc;">
+          <li>process the application</li>
+          <li>meet legal requirements</li>
+          <li>make improvements to this service</li>
+        </ul>
+      </div>
+      <div class="para">
+        <span class="text-16">Personal data is processed by our staff in theUK and the data is stored in the UK.</span>
```
Comment (noronhaa) [inline src/app/footer-nav/privacy.component.html]: whitespace needed: theUK -> the UK

File: src/app/footer-nav/cookies.component.html [hunk 6]
```diff
@@ -735,4 +735,289 @@ <h3 class="heading-medium">Cookies used to make the service more secure</h3>
...
+  <ccd-accordion [title]="'Family Public Law'" [lastAccordion]="true">
+    <div class="accordion-body">
+
+      <!-- Family Public Law Start -->
+      <div class="para">
+        <span class="text-16">A cookie is a small piece of data that’s stored on your computer, tablet, or
+          phone when you visit a website. Most websites need cookies to work properly.</span>
+      </div>
+
+      <h3 class="heading-medium">How cookies are used in this service</h3>
```
Comment (noronhaa) [inline src/app/footer-nav/cookies.component.html]: Check back at the content pdf doc, this needs to move to line 763

File: src/app/footer-nav/cookies.component.html [hunk 7]
```diff
@@ -735,4 +735,289 @@ <h3 class="heading-medium">Cookies used to make the service more secure</h3>
...
+          <li>temporarily store the answers you give</li>
+        </ul>
+      </div>
+      <div class="para">
+        <span class="text-16">
+          Find out more about <a href="http://www.aboutcookies.org/managing-cookies" target="_blank">
+            how to manage cookies</a>.</span>
+      </div>
+
+      <h3 class="heading-medium">To measure website usage</h3>
```
Comment (noronhaa) [inline src/app/footer-nav/cookies.component.html]: should be '1. To measure website usage' - need to add 1. prefix

File: src/app/footer-nav/cookies.component.html [hunk 8]
```diff
@@ -735,4 +735,289 @@ <h3 class="heading-medium">Cookies used to make the service more secure</h3>
...
+          <li>the pages you visit</li>
+          <li>how long you spend on each page</li>
+          <li>how you got to the service</li>
+          <li>what you click on while you’re visiting the service.</li>
+        </ul>
+      </div>
+      <div class="para">
+        <span class="text-16">We allow Google to use or share our
+            analytics data. You can find out more about how Google use this information in their
+            <a [routerLink]="'/privacy-policy'" target="_blank">privacy policy</a>.</span>
```
Comment (noronhaa) [inline src/app/footer-nav/cookies.component.html]: wrong privacy policy hyperlink

File: src/app/footer-nav/cookies.component.html [hunk 9]
```diff
@@ -735,4 +735,289 @@ <h3 class="heading-medium">Cookies used to make the service more secure</h3>
...
+        </div>
+        <div class="table-col-2">
+          <span class="text-16">Identifies you to the service</span>
+        </div>
+        <div class="table-col-3">
+          <span class="text-16">24 hours</span>
+        </div>
+      </div>
+
+      <h3 class="heading-medium">To turn our introductory message off</h3>
```
Comment (noronhaa) [inline src/app/footer-nav/cookies.component.html]: see pdf doc, needs '2.' prefix

File: src/app/footer-nav/cookies.component.html [hunk 10]
```diff
@@ -735,4 +735,289 @@ <h3 class="heading-medium">Cookies used to make the service more secure</h3>
...
+        </div>
+        <div class="table-col-2">
+          <span class="text-16">Saves a message to let us know that you've seen our cookie message</span>
+        </div>
+        <div class="table-col-3">
+          <span class="text-16">1 month</span>
+        </div>
+      </div>
+
+      <h3 class="heading-medium">To store the answers you’ve given during your visit (known as a ‘session’)</h3>
```
Comment (noronhaa) [inline src/app/footer-nav/cookies.component.html]: needs 3. prefix

File: src/app/footer-nav/cookies.component.html [hunk 11]
```diff
@@ -735,4 +735,289 @@ <h3 class="heading-medium">Cookies used to make the service more secure</h3>
...
+        </div>
+        <div class="table-col-2">
+          <span class="text-16">Used in the Money claims service to keeps track of your answers</span>
+        </div>
+        <div class="table-col-3">
+          <span class="text-16">When you close your browser</span>
+        </div>
+      </div>
+
+      <h3 class="heading-medium">To identify you when you come back to the service</h3>
```
Comment (noronhaa) [inline src/app/footer-nav/cookies.component.html]: needs '4.' prefix

File: src/app/footer-nav/cookies.component.html [hunk 12]
```diff
@@ -735,4 +735,289 @@ <h3 class="heading-medium">Cookies used to make the service more secure</h3>
...
+        </div>
+        <div class="table-col-2">
+          <span class="text-16">Used in the Money claim service to identify your preferred language</span>
+        </div>
+        <div class="table-col-3">
+          <span class="text-16">When you close your browser</span>
+        </div>
+      </div>
+
+      <h3 class="heading-medium">To make the service more secure</h3>
```
Comment (noronhaa) [inline src/app/footer-nav/cookies.component.html]: 5. prefix needed

File: src/app/footer-nav/cookies.component.html [hunk 13]
```diff
@@ -735,4 +735,289 @@ <h3 class="heading-medium">Cookies used to make the service more secure</h3>
...
+        <div class="table-col-2">
+          <span class="text-16">Helps us keep track of your session</span>
+        </div>
+        <div class="table-col-3">
+          <span class="text-16">When you close your browser</span>
+        </div>
+      </div>
+      <div class="table-row" style="height: 27px;">
+        <div class="table-col-1">
+          <span class="text-16">_csrf</span>
```
Comment (noronhaa) [inline src/app/footer-nav/cookies.component.html]: should be _csarf

File: src/app/footer-nav/cookies.component.html (hunk 8 above)
Comment (MKaramath) [reply src/app/footer-nav/cookies.component.html]: I couldn't see any issue here.  Can you please elaborate?

Comment (noronhaa) [reply src/app/footer-nav/cookies.component.html]: check back to the content doc attached to the story, You are supposed to hyperlink to the google privacy policy page not the moj link

https://policies.google.com/technologies/partner-sites?hl=en-GB&gl=uk

**Pull request merged**